app/forms/rating-schemas.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+import { zfd } from "zod-form-data";
+import { withZod } from "@remix-validated-form/with-zod";
+
+const rateJokeFormSchema = zfd.formData({
+  score: zfd.numeric(z.number().int().min(0).max(4)),
+});
```

Comment:
This is probably overkill, but I wanted to mess around with the validated forms and zod to make sure I understood how it all worked